changelog/TraitsParameters.dd

```diff
@@ -0,0 +1,81 @@
+Add `__traits(parameters)`
+
+If used inside a function, this trait yields a
+tuple of the parameters to that function for example:
```

Comment:
Could we make it usable in the return value of the function as well ?

Reply:
@RazvanN7 @maxhaton : This was still pending...

Reply:
You can't return tuples anyway.

Reply:
Sorry, but yeah the only way to do is to consume the trait inside an expression rather than returning it bare.

Reply:
You can easily compose. E.g.
```D
import std.stdio;

void main ()
{
 	auto x = foo(42, "Hello World");
    writeln(x);
}

struct MyTuple (T...)
{
 	T values;   
}

MyTuple!T foo(T...)(T args)
{
 	return typeof(return)(args);   
}
```

Reply:
That's what I had in mind in my comment.